src/aead/aead.rs

```diff
@@ -77,11 +77,14 @@ impl OpeningKey {
 
 /// Authenticates and decrypts (“opens”) data in place. When
 ///
-/// The input may have a prefix that is `in_prefix_len` bytes long; any such
+/// `nonce` must be unique for every use of the key to seal data; it must be
```

Comment:
> Since this is the open_in_place method, maybe the nonce should be explained as "the same nonce as was used for sealing"?

The caller of `open_in_place()` generally cannot know the value of the nonce that was used for sealing; it basically can only know that it did the sealing itself. The nonce used in `open_in_place()` will be whatever nonce was sent by the peer if explicit nonces are used. In the good-but-rare cases where implicit nonces are used, the nonce will be whatever the protocol being implemented specifies it to be. In other words, the caller of `open_in_place()` doesn't have any real flexibility in choosing the nonce in any protocol.

When explicit nonces are used, the caller of `seal_in_place()` does have a lot more flexibility. Maybe we should advise them to ensure they always use a nonce that is unique for the given key, and to not to use random nonces since the nonce length is too small to satisfy the uniqueness requirement with random nonces. But providing more advice than that seems kind of out-of-scope since it's a protocol design issue.